examples/history/value.json

```diff
@@ -0,0 +1,34 @@
+{
+  "document": {
+    "nodes": [
+      {
+        "kind": "block",
+        "type": "paragraph",
+        "nodes": [
+          {
+            "kind": "text",
+            "leaves": [
+              {
+                "text": "Editor maintains its own history stack. For custom components and changes, history operations are automatically handled, no need implementing their undo/redo methods by yourself!"
```

Comment:
For the example text, can we go with:

> Slate editors save all changes to an internal "history" automatically, so you don't need to implement undo/redo yourself. And the editor automatically binds to the browser's default undo/redo keyboard shortcuts.

> Try it out for yourself! Make any changes you'd like then press "cmd+z".

Thanks!

Reply:
Fixed!